about-pses/example-pse.md

```diff
@@ -117,6 +136,14 @@ class TestPython1(unittest.TestCase):
 ##### !end-tests
 ##### !explanation 
 
+The submission for this prompt only saves your input. The tests are not evaluated, nor is the syntax even checked. Please take care with your own syntax and how the tests are written, as _anything_ we submit will be marked "Correct."
```

Comment:
I feel like this could be worth moving to the prompt itself, since they'll only see this reminder to check syntax after they submit.